### DISCUSSION TITLE
Keep document centered when scrolling

### DISCUSSION THREAD

**Discussion by PauLuke** [ahrm/sioyek#269] [Q&A] [answered]
When I use my laptop's touchpad to scroll I always end up taking the document off-center, so I was wondering if there's a way to disable horizontal scrolling or something, so the document never gets off-center while scrolling.

**Comment by ahrm** [comment, marked as answer]
Yes, there is a `toggle_horizontal_scroll_lock` command.

**Reply by qingshuizheng** [reply]
Hi @ahrm ahrm

Are we able to set horizontal_scroll off as default, maybe somewhere in configs? 

I prefer to have it as default without having to toggle it. And currently the way I do it is to set `horizontal_move_amount    0` in prefs_user.config.

**Reply by qingshuizheng** [reply]
just watched the tutorial again and learnt that `startup_commands toggle_horizontal_scroll_lock` should do the trick. 
sorry for missing that.
